Tools/Scripts/webkitpy/port/apple.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, host, port_name, **kwargs):
 
         port_name = port_name.replace('-wk2', '')
         self._version = self._strip_port_name_prefix(port_name)
+        self.supports_localhost_aliases = True
```

Comment:
```suggestion
        self.supports_localhost_aliases = False
```

if the intention is still to land this still disabled?